src/main/java/wooteco/subway/line/repository/LineDao.java

```diff
@@ -0,0 +1,60 @@
+package wooteco.subway.line.repository;
+
+import org.springframework.util.ReflectionUtils;
+import wooteco.subway.line.Line;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class LineDao implements LineRepository {
```

Comment:
```suggestion
public class InMemoryLineDao implements LineDao {
```

LineDao를 추상화했고 그 구현체에서 메모리에 데이터를 저장하고 있으니 위와 같이 표현하는게 적절해보이네요